manifests/manifest-base.yml

```diff
@@ -2,8 +2,10 @@
 buildpack: go_buildpack
 memory: 256M
 services:
-  - deck-ups
+- deck-ups
+- deck-redis
```

Comment:
Should we be calling these `dashboard` now? Or maybe this is not the right time to make a name change, that's fine too.